src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionTestResultConsentViewModel.swift

```diff
@@ -107,10 +95,23 @@ class ExposureSubmissionTestResultConsentViewModel {
 					cells:[
 						.space(height: 50)
 					]
-					
+
 				)
 			)
 		}
 	}
-	
+
+	// MARK: - Private
+
+	private let consentSwitch = UISwitch()
```

Comment:
Sorry that I just notice it when you moved it. But a UISwitch in a ViewModel? O.o

Reply:
Looks like we need an extra DynamicTableViewCell with separated view model here. Shall we create a new Task for it?